src/functionaltest/java/com/ericsson/ei/frontend/SubscriptionHandlingFunctionality.java

```diff
@@ -45,20 +50,68 @@ public void testSubscription() throws Exception {
         assert (new WebDriverWait(driver, 10)
                 .until((webdriver) -> subscriptionPage.presenceOfHeader(subscriptionHeaderID)));
 
-        // // Press "Reload" button and verify that two subscriptions with names
-        // "Subscription1" and "Subscription2" are present
+        // Press "Reload" button without enabling LDAP and verify that two subscriptions
+        // with names
+        // "Subscription1" and "Subscription2" are present AND there exists "edit" and
+        // "delete buttons" for unauthorized user "ABCD"
         String response = this.getJSONStringFromFile(RELOAD_TEST_FILE_PATH);
+        String deletePath = "//tr[td[contains(.,'Subscription1')]]/td/button[contains(text(),'Delete')]";
+        String editPath = "//tr[td[contains(.,'Subscription1')]]/td/button[contains(text(),'Edit')]";
+        String viewPath = "//tr[td[contains(.,'Subscription1')]]/td/button[contains(text(),'View')]";
         subscriptionPage.clickReload(response);
         assert (new WebDriverWait(driver, 10)
                 .until((webdriver) -> ((driver.getPageSource().contains("Subscription1")))));
         assert (new WebDriverWait(driver, 10)
                 .until((webdriver) -> ((driver.getPageSource().contains("Subscription2")))));
 
+        assert (subscriptionPage.buttonExist(deletePath) == true);
+        assert (subscriptionPage.buttonExist(editPath) == true);
+        assert (subscriptionPage.buttonExist(viewPath) == true);
+
+        // Given LDAP is enabled, "Reload" subscriptions and then click subscription
+        // page with LDAP enabled with unauthorized user names
+        // Verify that subscriptions exists but only with "View" button
+        String responseSub = this.getJSONStringFromFile(RELOAD_TEST_FILE_PATH_LDAP);
```

Comment:
I think you can write out the variable names, becomes way easier to read :) And i think a more clear variable name if RELOAD_TEST_FILE_PATH_LDAP was renamed to SUBSCRIPTION_FOR_UPLOAD_LDAP_FILEPATH(see my other comment) then this variable could be subscriptionForUploadLdap. 

Right now its hard to couple responseSub with RELOAD_TEST_FILE_PATH_LDAP without reading this line.
I think in general, look through this file, there are more similar cases.